cli/main.rs

```diff
@@ -168,7 +168,7 @@ fn display_schema(
             table_name
         ),
         None => String::from(
-            "SELECT sql FROM sqlite_schema WHERE type IN ('table', 'index') AND name NOT LIKE 'sqlite_%' ORDER BY type, name"
+            "SELECT sql FROM sqlite_schema WHERE type IN ('table', 'index') AND name NOT LIKE 'sqlite_%'"
```

Comment:
Why did you drop the ORDER BY?

Reply:
Because `test-sqlite` fails if they are ordered according to name. I made an (unverified) assumption that they are returned in insertion order

Reply:
I originally had the test expected output match what `limbo` outputted, but `test_sqlite` failed: `limbo` was outputting `CREATE TABLE products` first (because p < u), while `sqlite3` was outputting `CREATE TABLE users` first.